ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,1094 @@
+====
+
+[#_running_on_openshift_run_the_quickstart_with_productname_operator]
+==== Running on OpenShift: Run the Quickstart with {productName} Operator
+
+Now it's possible to deploy the`CustomerResource` of {productName} Operator.
+The `CustomerResource` `yaml` definition contains information for the {productName} Operator starting
+the application pods for the `client` and for the `server`.
+
+[NOTE]
+====
+If you follow the OpenShift prerequisite step then you work under OpenShift project {artifactId}-project.
+Verify that the `applicationImage` value under `${PATH_TO_QUICKSTART_DIR}/ejb-txn-remote-call/client/client-cr.yaml`
+`${PATH_TO_QUICKSTART_DIR}/ejb-txn-remote-call/server/server-cr.yaml` matches with the project name.
+
+You can check with `oc get image`.It should show you a similar output to this.You should see the {artifactId}-project in the name.
+
+[source,plain,options="nowrap",subs="+quotes,attributes+"]
+----
+NAME                    IMAGE REPOSITORY                                                                                     TAGS     UPDATED
+client                  default-route-openshift-image-registry.apps-crc.testing/{artifactId}-project/client                  latest   2 minutes ago
+client-build-artifacts  default-route-openshift-image-registry.apps-crc.testing/{artifactId}-project/client-build-artifacts  latest   8 minutes ago
+server                  default-route-openshift-image-registry.apps-crc.testing/{artifactId}-project/server                  latest   2 minutes ago
+server-build-artifacts  default-route-openshift-image-registry.apps-crc.testing/{artifactId}-project/server-build-artifacts  latest   9 minutes ago
+----
+====
+
+Before starting the application we need to add `view` permissions for the default system account.
+That's needed by `KUBE_PING` protocol which is the protocol used to establish the cluster of {productName} nodes
+with servers `server2` and `server3`.
+For the developer purposes we add the view role for the default service account with the following command.
+
+[source,sh]
+----
+oc policy add-role-to-user view system:serviceaccount:$(oc project -q):default -n $(oc project -q)
+----
+
+Now you may create the `CustomerResource` which will be managed by {productName} Operator.
+
+[source,sh,options="nowrap"]
+----
+cd ${PATH_TO_QUICKSTART_DIR}
+
+# deploying client definition, one replica, PostgreSQL database has to be available
+oc create -f ejb-txn-remote-call/client/client-cr.yaml
+# deploying server definition, two replicas, PostgreSQL database has to be available
+oc create -f ejb-txn-remote-call/server/server-cr.yaml
+----
+
+As the result there should be a running the quickstart application pods when executed `oc get pod`.
+At the side of the application pods we expect to have running the {productName} Operator pod,
+and the PostgreSQL database pod as the application will connect to it.
```

Comment:
If I understood this paragraph correctly, how about rephrasing it as:

```
If these commands are successful, the `oc get pod` command should show
all the pods required for the quickstart, namely the quickstart client and two
server pods, the WildFly Operator pod and the PostgreSQL database pod
that the application connects to.
```

Reply:
yes, you do. I changed it. thanks
+1